### ISSUE TITLE
Unexpected response code: 413 (Value exceeds 524288 byte limit)

### ISSUE THREAD

**Issue by anentropic** [docker/classicswarm#1941]
(previously posted as https://github.com/docker/compose/issues/3041 but apparently it's a swarm issue)

So I've been going round and round trying to get my swarm working

Thought I was getting close but now when I `docker-compose up` I get the following error:

```
... lots of --verbose logging ...
compose.service._containers_have_diverged: oceania_nginx-conf_1 has diverged: e986cd8a43c1825f73d62af8ac5fd0c467d185f8e06d1cd89394ff11fcad86ed != e48efc42c1e29a8ba5d862593cd23da42004d996959411d34bcc106629144c09
compose.cli.verbose_proxy.proxy_callable: docker inspect_network <- (u'oceania_oceania')
compose.network.ensure: Creating network "oceania_oceania" with driver "overlay"
compose.cli.verbose_proxy.proxy_callable: docker create_network <- (ipam=None, driver='overlay', options=None, name=u'oceania_oceania')
ERROR: compose.cli.main.main: 500 Internal Server Error: pool configuration failed because of Unexpected response code: 413 (Value exceeds 524288 byte limit)
```

seems to be in the step to create the overlay network

...

`docker-machine restart` fixed it (I restarted both the master and the consul node in my swarm, not sure which fixed it)

**Comment by ulnneverknow** [comment]
This is the exact same problem that @auzias was having with #1931 . I had a similar problem which I solved by restarting the docker daemons and everything worked again.

**Comment by michelvocks** [comment]
+1 I have the same issue

**Comment by auzias** [comment]
@michelvocks did you try to reboot your daemon[s]?

> Thanks to the open letter of GH users and GH new feature, there is [no need to comment for simple reaction](http://wptavern.com/github-now-supports-emoji-reactions-for-pull-requests-issues-and-comments)

**Comment by HadesArchitect** [comment]
Relates to https://github.com/hashicorp/consul/issues/1682.

**Comment by bennetimo** [comment]
This seems to be more related to overlay networking using external Consul k/v store from what I've seen. The frame size limit seems to come from Consul. Added a comment here: https://github.com/docker/compose/issues/3041#issuecomment-252572280.

One other thing I've noticed when hitting this trying to create an overlay network on a node, call it A:

- Creating it through A hangs/eventually fails with this error 413. Docker daemon restart fixes it and allows it to go through subsequently
- Creating it through a different node B in the cluster (that has not previously failed with error 413) creates the network successfully.
